### ISSUE TITLE
Pokémon Omega Ruby/Alpha Saphire - Multiple thick bars appear behind starter Pokémons

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#2229]
The thick bars will remain appear even when resizing window and *scaled resolution* is disabled initially.
Sometimes it just auto disappear for no apparent reason or rarely it does not even appear at all, the issue happen at completely random time.

Also the thick bars will also appear when *scaled resolution* is enabled but will disappear when resizing the window later.

Subsequent toggling on/off *scaled resolution* will make the visual glitch disappear (appear as intended)

This is what happen when scaled resolution is turned off initially
![Present thick bar](https://cloud.githubusercontent.com/assets/18496868/20690779/8e1c03e8-b607-11e6-8557-1ba71039e0f6.png)
This is the result of turning on scaled resolution and subsequent toggle on/off scaled resolution
![Absent thick bar](https://cloud.githubusercontent.com/assets/18496868/20690801/a07dd89a-b607-11e6-8284-da4733dfd089.png)

This log appear when reaching this scene
```
HW.GPU <Critical> video_core\renderer_opengl\gl_shader_gen.cpp:AppendSource:58: Unhandled texture type 1
HW.GPU <Critical> video_core\renderer_opengl\gl_shader_gen.cpp:AppendSource:58: Unhandled texture type 1
HW.GPU <Critical> video_core\renderer_opengl\gl_shader_gen.cpp:AppendSource:58: Unhandled texture type 1
HW.GPU <Critical> video_core\renderer_opengl\gl_shader_gen.cpp:AppendSource:58: Unhandled texture type 1
HW.GPU <Critical> video_core\renderer_opengl\gl_shader_gen.cpp:AppendSource:58: Unhandled texture type 1
HW.GPU <Critical> video_core\renderer_opengl\gl_shader_gen.cpp:AppendSource:58: Unhandled texture type 1
```
Tested this on the latest commit as of 3174bfd

This one I can confirm is not a driver issue as others experience this issue as well and my _OpenGL_ version is up-to-date. See #2041

**OS:** _Windows 10 build 14393.447 (codename Redstone 1)_
**GPU:** _nVidia GT 610 2GB DDR3 (Asus)_
**Driver:** _Version 375.95_
**OpenGL:** _Version 4.5_

**Comment by NicolaiVdS** [comment]
pleas provide following:

- os
- specs
-what build (can be found on the title bar should be something like 'Citra | master-3174bfd' only need the '3174bfd' part

**Comment by Dragios** [comment]
Dude, this is a visual glitch, don't need OS version and specs as all will encounter the same problem.

**Comment by yuriks** [comment]
Knowing at least your GPU manufacturer is very helpful for graphics issues, because of driver issues/bugs.

**Comment by NicolaiVdS** [comment]
@Dragios first off all by os i mean (windows, linux or MacOs) second by specs you declare what GPU if you have one, this way we know what openGL version your setup is running and as last we need to know what build you are using so we can recreate the error you so we can fix it 

-Dude

**Comment by Dragios** [comment]
This 1 I can confirm is not a driver issue as others experience this issue as well and my OpenGL version is up-to-date. See #2041
I just created a new issue as @JayFoxRox wanted every issue to be separated for easy label issues and also make it easier to find out what's being discussed on.

Anyway I will just provide all the details needed nevertheless
OS: Windows 10 build 14393.447 (codename Redstone 1)
GPU: nVidia GT 610 2GB DDR3 (Asus)
Driver: Version 375.95
OpenGL: Version 4.5

**Comment by RavenHome1** [comment]
duplicate of https://github.com/citra-emu/citra/issues/2220

**Comment by wwylele** [comment]
@RavenHome1 no they are different issues. At least they are different in appearance

**Comment by NicolaiVdS** [comment]
Also happens in Pokemon Sun/Moon ![pkmnsun](http://i.imgur.com/NQatPgw.png)

**Comment by JayFoxRox** [comment]
@NicolaiVdS As this appears to be a different scene (and even game..), different appearance and unconfirmed to be the same bug, please open a new issue and explain how to reach those spots in the game. Also test if this also is related to scaled resolution.
If you think this should be the same bug just reference it and once we fix one of them we'll just merge issues.
"This is probably the same"-posts often cause meta-issues or send us down an offtopic-spiral.

**Comment by NWPlayer123** [comment]
It seems this is still happening (regression? @Hexagon12 says it was fixed)
Tested on HEAD-0e8c25f, default settings, just ran 3dsutils and put the needed files in nand
I haven't been able to narrow it down, seems like it may be texture invalidation or a timing issue? it's only ever happened after I'm already running it, in the first battle with the Poochyena, and without Stopping, just load the cxi again

**Comment by jroweboy** [comment]
@NWPlayer123 the texture cache rework isn't merged. try it out on canary to see if it works there.

**Comment by NWPlayer123** [comment]
@jroweboy played around with it, couldn't replicate, I guess it's fixed, good to know 👍 I'll be sure to check where stuff was merged next time
dunno if this should be closed, could probably remain open until it's merged into nightly but I doubt there'll be a regression

**Comment by wwylele** [comment]
Fixed by #3281 as reported so closing. Feel free to reopen if it is not the case.
